tests/unit/test_slack_notification.py

```diff
@@ -0,0 +1,75 @@
+import os
+import unittest
+from mock import patch, Mock
+import sys
+import mock_import
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../../src/slackNotification')))
+from requests.exceptions import HTTPError
+import slack_notification
+
+
+class TestSlackNotification(unittest.TestCase):
+
+    def _mock_response(
+            self,
+            status=200,
+            raise_for_status=None):
+        mock_resp = Mock()
+        # mock raise_for_status call w/optional error
+        mock_resp.raise_for_status = Mock()
+        if raise_for_status:
+            mock_resp.raise_for_status.side_effect = raise_for_status
+        mock_resp.status_code = status
+        return mock_resp
+
+    def test_send_to_slack_response_success(self):
+        factory_patch = patch('slack_notification.requests')
```

Comment:
can this be converted to use the @patch annotation?

Reply:
I wanted to use @patch instead of the function but turns out the remaining tests were using patch this way. wanted to maintain uniformity. would you still like me to make this change?

Reply:
yes go ahead and make the change

Reply:
@namitad updated